avm/res/consumption/budget/tests/e2e/forecasted/main.test.bicep

```diff
@@ -0,0 +1,37 @@
+targetScope = 'subscription'
+
+metadata name = 'Using `thresholdType` `Forecasted`'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
```

Comment:
Description needs updating (and then the readme a refresh too)

Reply:
done